sql-select/cheap-to-rent.sql

```diff
@@ -0,0 +1,4 @@
+select "filmId",
+       "title"
+  from "films"
+where "rentalRate" < 1;
```

Comment:
Please make sure you're only selecting 50 films:

---

![image](https://user-images.githubusercontent.com/16143771/186278773-e94996ed-d573-4e38-8d6e-4b47f7414656.png)